web/src/components/njwds-layout/SidebarPageLayout.tsx

```diff
@@ -23,45 +24,52 @@ export const SidebarPageLayout = ({
   outlineBox = true,
   stackNav,
   belowBoxComponent,
+  titleOverColumns,
 }: SidebarPageLayoutProps): ReactElement => {
   const isLargeScreen = useMediaQuery(MediaQueries.desktopAndUp);
   const { Config } = useConfig();
 
   const backButton = (
-    <Link href={ROUTES.dashboard} passHref>
-      <a
-        href={ROUTES.dashboard}
-        data-testid="back-to-dashboard"
-        className="usa-link fdr fac margin-top-3 margin-bottom-3 usa-link-hover-override desktop:margin-top-0 desktop:margin-bottom-4"
-        onClick={analytics.event.task_back_to_roadmap.click.view_roadmap}
-      >
-        <div className="circle-3 bg-accent-cool-darkest icon-blue-bg-color-hover">
-          <Icon className="text-white usa-icon--size-3">arrow_back</Icon>
-        </div>
-        <div className="margin-left-2 margin-y-auto text-accent-cool-darkest font-sans-xs usa-link text-blue-color-hover">
-          {Config.taskDefaults.backToRoadmapText}
-        </div>
-      </a>
-    </Link>
+    <nav>
+      <Link href={ROUTES.dashboard} passHref>
+        <a
+          href={ROUTES.dashboard}
+          data-testid="back-to-dashboard"
+          className="usa-link fdr fac margin-top-3 margin-bottom-3 usa-link-hover-override desktop:margin-top-0 desktop:margin-bottom-4"
+          onClick={analytics.event.task_back_to_roadmap.click.view_roadmap}
+        >
+          <div className="circle-3 bg-accent-cool-darkest icon-blue-bg-color-hover">
+            <Icon className="text-white usa-icon--size-3">arrow_back</Icon>
+          </div>
+          <div className="margin-left-2 margin-y-auto text-accent-cool-darkest font-sans-xs usa-link text-blue-color-hover">
+            {Config.taskDefaults.backToRoadmapText}
+          </div>
+        </a>
+      </Link>
+    </nav>
   );
 
   return (
     <>
       <div className={`usa-section padding-top-0 desktop:padding-top-6`}>
         <div className="grid-container-widescreen desktop:padding-x-7">
+          {isLargeScreen && backButton}
+
+          {isLargeScreen && titleOverColumns}
           <div className="grid-row grid-gap">
             <div className="desktop:grid-col-9">
               {!isLargeScreen && (
                 <div>
                   {backButton}
+                  {titleOverColumns}
                   {stackNav && navChildren}
                 </div>
               )}
               <div
                 className={
                   outlineBox
                     ? "border-1px border-base-light usa-prose minh-40 padding-4 radius-lg"
-                    : "minh-40 padding-y-205"
+                    : "minh-40 padding-y-"
```

Comment:
[pebble] I don't believe `padding-y-` is a valid USWDS class, but I'm still getting familiar with it, so I may be wrong here.